src/Components/Main/Pages/ProfilePage/ProfileInfo/ProfileInfoContainer.js

```diff
@@ -2,26 +2,47 @@ import { ProfileInfo } from './ProfileInfo';
 import { useCallback } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { useMounted } from '../../../../../hooks/useMounted';
-import { updateStatus } from '../../../../../redux/thunks/profile';
-import { isFetchingStatusSelector } from '../../../../../redux/selectors/loading';
+import { updateData, updateStatus, updatePhoto } from '../../../../../redux/thunks/profile';
+import {
+  isFetchingStatusSelector,
+  isFetchingPhotoSelector,
+  isFetchingDataSelector
+} from '../../../../../redux/selectors/loading';
 import { statusSelector } from '../../../../../redux/selectors/profile';
 
-export const ProfileInfoContainer = ({profile}) => {
+export const ProfileInfoContainer = ({profile, isOwner}) => {
   const status = useSelector(statusSelector);
   const isFetchingStatus = useSelector(isFetchingStatusSelector);
+  const isFetchingPhoto = useSelector(isFetchingPhotoSelector);
+  const isFetchingData = useSelector(isFetchingDataSelector);
 
   const dispatch = useDispatch();
   const updateStatusCallback = useCallback(
     (status) => dispatch(updateStatus(status)),
     [dispatch]
   );
+  const updatePhotoCallback = useCallback(
+    (photo) => dispatch(updatePhoto(photo)),
+    [dispatch]
+  );
+  const updateDataCallback = useCallback(
+    (profileData, setStatus, setSubmitting, setEditModeData) => {
+      return dispatch(updateData(profileData, setStatus, setSubmitting, setEditModeData));
```

Comment:
Do you really need "return" here? =)

Reply:
No, this is not necessary.
I have removed the "return" !